lib/smart_proxy_ansible/runner/ansible_runner.rb

```diff
@@ -70,7 +71,7 @@ def kill
 
       def close
         super
-        FileUtils.remove_entry(@root) if @tmp_working_dir && Dir.exist?(@root)
+        FileUtils.remove_entry(@root) if @tmp_working_dir && Dir.exist?(@root) if @cleanup_working_dirs
```

Comment:
Double `if` feels weird
```suggestion
        FileUtils.remove_entry(@root) if @tmp_working_dir && Dir.exist?(@root) && @cleanup_working_dirs
```